tasks.py

```diff
@@ -399,3 +400,47 @@ def calculate_accuracy(self, predictions, labels):
         correct = bin_labels.sum().float().item()
         return correct / len(labels)
 
+
+class SentimentAnalysis(Task):
+    NAME = 'SentimentAnalysis'
+
+    def __init__(self, fn_tokenizer=bert_tokenizer):
+        self.num_classes = 2
+        self.classifier = MLPClassifier(target_dim=self.num_classes)
+        self.criterion = CrossEntropyLoss()
+        self.fn_tokenizer = fn_tokenizer
+
+    def get_iter(self, split, tokenizer, batch_size=16, shuffle=False, random_state=1, max_length=64):
+        """
+        Returns an iterable over the single
+        Args:
+            split: train/dev/test
+        Returns:
+            Iterable for the specified split
+        """
+        # current iter will have only two classes; we could extend it to have more
+        df = pd.read_csv('data/tweets_output.txt',header=None, sep='\t', names=['ID1', 'ID2', 'label','sentence'])
+        df = df[df.label != 'neutral']
+        df = df[df.label != 'objective']
+        df = df[df.label != 'objective-OR-neutral']
+
+        sentences = df.sentence.values
+        labels = np.where(df.label.values == 'positive', 1, 0)
+
+        input_ids, attention_masks = self.fn_tokenizer(sentences, tokenizer, max_length=max_length)
+        labels = torch.tensor(labels)#.unsqueeze(1)
+
+        return make_dataloader(input_ids, labels, attention_masks, batch_size, shuffle)
```

Comment:
Depending on what PR goes to master first this might need to pass as 1st parameter an ID for the dataset. (See #20)

Reply:
As #20 was merged, you'll need to pass a dataset identifier as first parameter to make_dataloader as in https://github.com/baleato/atcs-project/blob/master/tasks.py#L187.

Please add the task name as dataset identifier:
```python
return make_dataloader(self.NAME, input_ids, labels, ...
```

Reply:
Hey, added the task name in my latest commit.